src/pages/settings/Preferences/PreferencesPage.js

```diff
@@ -0,0 +1,137 @@
+        },
+    };
+    const localesToLanguages = {
+        en: {
+            value: 'en',
+            label: props.translate('languagesPage.languages.english'),
+        },
+        es: {
+            value: 'es',
+            label: props.translate('languagesPage.languages.spanish'),
+        },
+    };
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <HeaderWithCloseButton
+                title={props.translate('common.preferences')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <ScrollView style={[styles.flex1, styles.mt3]}>
+                <View style={styles.mb6}>
+                    <Text style={[styles.textLabelSupporting, styles.mb2, styles.ml8, styles.mr8]} numberOfLines={1}>
+                        {props.translate('common.notifications')}
+                    </Text>
+                    <View style={[styles.flexRow, styles.mb4, styles.justifyContentBetween, styles.ml8, styles.mr8]}>
+                        <View style={styles.flex4}>
+                            <Text>
+                                {props.translate('preferencesPage.receiveRelevantFeatureUpdatesAndExpensifyNews')}
+                            </Text>
+                        </View>
+                        <View style={[styles.flex1, styles.alignItemsEnd]}>
+                            <Switch
+                                isOn={lodashGet(props.user, 'isSubscribedToNewsletter', true)}
+                                onToggle={User.updateNewsletterSubscription}
+                            />
+                        </View>
+                    </View>
+                    <MenuItemWithTopDescription
+                        shouldShowRightIcon
+                        title={priorityModes[props.priorityMode].label}
+                        description={props.translate('priorityModesPage.priorityMode')}
+                        onPress={() => Navigation.navigate(ROUTES.SETTINGS_PRIORITY_MODES)}
+                    />
+                    <MenuItemWithTopDescription
+                        shouldShowRightIcon
+                        title={localesToLanguages[props.preferredLocale].label}
+                        description={props.translate('languagesPage.language')}
+                        onPress={() => Navigation.navigate(ROUTES.SETTINGS_LANGUAGES)}
+                    />
+
+                    {/* If we are in the staging environment then we enable additional test features */
+                        _.contains([CONST.ENVIRONMENT.STAGING, CONST.ENVIRONMENT.DEV], props.environment)
+                        && (
+                            <View style={[styles.ml8, styles.mt6, styles.mr8]}>
+                                <TestToolMenu />
+                            </View>
+                        )
+                    }
```

Comment:
Possibly missing a set of brackets?

```suggestion
                    {/* If we are in the staging environment then we enable additional test features */}
                    {
                        _.contains([CONST.ENVIRONMENT.STAGING, CONST.ENVIRONMENT.DEV], props.environment)
                        && (
                            <View style={[styles.ml8, styles.mt6, styles.mr8]}>
                                <TestToolMenu />
                            </View>
                        )
                    }
```